toolchain/lowering/lowering_function_context.h

```diff
@@ -21,6 +24,26 @@ class LoweringFunctionContext {
   explicit LoweringFunctionContext(LoweringContext& lowering_context,
                                    llvm::Function* function);
 
+  // Returns a basic block corresponding to the start of the given semantics
+  // block.
+  auto GetBlock(SemanticsNodeBlockId block_id) -> llvm::BasicBlock*;
```

Comment:
WDYT if we used references to emphasize non-nullness, including things like GetValue?